addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -246,8 +246,15 @@ $input-placeholder-color: mix($input-bg, $input-color) !default;
 
 $input-disabled-bg: mix($input-color, $input-bg, 10%) !default;
 
+$form-select-indicator-color: $input-color !default;
+
 $form-check-input-border: ($input-border-width or 1px) solid $input-border-color !default;
-$form-range-track-bg: $light !default;
+$form-check-input-checked-color: color-contrast(o-color('primary')) !default;
+$form-range-track-bg: mix($input-color, $body-bg, 15%) !default;
+$form-range-thumb-bg: o-color('primary') !default;
+
+$form-switch-color: $input-color !default;
+$form-switch-checked-color: $form-check-input-checked-color !default;
```

Comment:
Feels like some of these should in the web bootstrap_overridden, not the website one. Anything that overrides the frontend should be:
- In website if it depends on a website-only variable
- Otherwise in portal if it depends on a portal-only variable
- Otherwise in web_editor if it depends on a web_editor-only variable
- Otherwise in web if it depends on nothing / bootstrap variable (like the first one here for example: "the form select indicator is the input color" => this is not something that should be website specific.

Reply:
Indeed ! I moved the variables in `web/bootstrap_overriden_frontend.scss` so that it works without installing website. 👍